fms/models/llama.py

```diff
@@ -195,6 +195,16 @@ def __init__(
             ntk_scaling=self.config.ntk_scaling,
             max_seq_len=self.config.max_expected_seq_len,
         )
+        # RoPE init
+        if isinstance(self.distributed_strategy, UniformModelParallelStrategy):
```

Comment:
if we do this in the constructor I don't think it needs to be in the reset_parameters code too

Reply:
Good point, I removed it from reset_parameters (as we should be calling this all the time)